svelte4fix.js

```diff
@@ -0,0 +1,49 @@
+const { readdirSync, readFileSync, writeFileSync } = require("fs");
```

Comment:
Probably don't need to check this file in

Reply:
@splimter just curious if you were aware of `svelte-migrate` which should migrate types. Was there something missing from it or a bug or anything like that?

Reply:
Hi @benmccann unfortunately no i was not aware of `svelte-migrate`.

Reply:
@splimter  do we need to consider svelte-migrate or is the necessary work already done? 
Thanks!